README.md

```diff
@@ -1,149 +1,86 @@
-# Getting started example for Mbed OS
+# Blinky Mbed OS Example
 
-This guide reviews the steps required to get Blinky with the addition of dynamic OS statistics working on an Mbed OS platform. (Note: To see a rendered example you can import into the Arm Online Compiler, please see our [quick start](https://os.mbed.com/docs/mbed-os/latest/quick-start/online-with-the-online-compiler.html#importing-the-code).)
```

Comment:
That link shows how to import the project. Someone reading this README already has the project because the README is in the project.
If someone is reading on GitHub therefore he will clone the project as it is done with GitHub.

Reply:
Could we replace it with a link here: https://os.mbed.com/docs/mbed-os/latest/quick-start/index.html

So anyone who uses the code as an entry point has a way to the docs?

Reply:
[Done](https://github.com/ARMmbed/mbed-os-example-blinky/pull/192/commits/2f96047ab8eb914c0ce2ee6805a598a1df3445e4).